src/main/java/seedu/address/logic/commands/FindPropertyCommand.java

```diff
@@ -19,10 +19,11 @@
             + "the specified keywords (case-insensitive) and/or by giving a price and search for properties "
             + "with asking prices above or below that amount, then displaying them as a list with index "
             + "numbers.\n"
-            + "Parameters: [KEYWORD]... [pl/UPPER_PRICE_LIMIT] [pm/LOWER_PRICE_LIMIT] \n"
+            + "Parameters: [KEYWORD]... [pl/UPPER_PRICE_LIMIT] [pm/LOWER_PRICE_LIMIT] [t/TAG] \n"
```

Comment:
I think we should be consistent with the add property command and use `[t/TYPE]` to indicate that the filter is for property type